src/views/templates/utils/constants.ts

```diff
@@ -27,3 +27,16 @@ export type SOURCE_OPTIONS_IDS =
   | typeof SOURCE_TYPES.uploadSource;
 
 export const SUPPORT_URL = 'https://access.redhat.com/articles/4234591';
+
+// t('Desktop')
+// t('Server')
+// t('High performance')
+// t('Other')
```

Comment:
you don't need `Other` because it's not a string in `workloadProfiles`

Reply:
you do need it in https://github.com/kubevirt-ui/kubevirt-plugin/pull/477/files#diff-c05bbb0bae554062323840c4381c34d6cc141eafbf6d0b2b406462c5f5a774b5R27 because it's the fallback value if translation key is missing in `workloadProfiles`, so maybe add a comment
``` ts
// Other is the default value in case translation key is missing in workloadProfiles
// t('Other')
```

Reply:
you should translate "Other" you can do it here 
or near
https://github.com/kubevirt-ui/kubevirt-plugin/pull/477/files#diff-c05bbb0bae554062323840c4381c34d6cc141eafbf6d0b2b406462c5f5a774b5R27
where it is defined

I like to translating it here, because it's part of the strings describing a workload, `['Desk ... , 'Other']`, it also makes sense to tranlate it near https://github.com/kubevirt-ui/kubevirt-plugin/pull/477/files#diff-c05bbb0bae554062323840c4381c34d6cc141eafbf6d0b2b406462c5f5a774b5R27 because it's defined there.